index.d.ts

```diff
@@ -13,9 +13,9 @@ declare module 'launchdarkly-node-server-sdk-dynamodb' {
 
   /**
    * Create a feature flag store backed by DynamoDB.
-   * 
+   *
    * @param tableName The table name in DynamoDB (required). The table must already exist.
-   *   See: https://docs.launchdarkly.com/sdk/features/storing-data/dynamodb
+   *   See: https://docs.launchdarkly.com/sdk/features/storing-data/dynamodb#nodejs-server-side
```

Comment:
@ember-stevens @bwoskow-ld I'm not sure this is a change we want to make. The topic relevant to this doc comment isn't just specifically "how to use the Node integration for DynamoDB" but "how the DynamoDB integrations work in general"— all of which is at the _top_ of that page, whereas this link will take you down to a subsection with Node code samples that are somewhat redundant if you're already looking at the API docs here.

Reply:
Works for me.